src/py42/sdk/queries/alerts/filters/alert_filter.py

```diff
@@ -27,22 +27,39 @@ def not_contains(cls, value):
 
 
 class DateObserved(QueryFilterTimestampField):
+    """Class that filters alerts based on the timestamp the event was observed."""
```

Comment:
Kinda related to another comment I made...

Maybe like `...based on the timestamp's observation date.` or `...based on when our systems observed the event.`